packages/tests/package.json

```diff
@@ -73,6 +68,8 @@
     "@libp2p/interface-peer-discovery-compliance-tests": "^2.0.5",
     "@libp2p/interface-peer-id": "^2.0.1",
     "@types/chai": "^4.3.4",
+    "@types/dockerode": "^3.3.15",
+    "dockerode": "^3.3.5",
```

Comment:
Technically should be in `dependencies` because it's use by `nwaku.ts` but does not really matter in a private package.